lib/services/service_brokers/v2/catalog_plan.rb

```diff
@@ -69,7 +71,8 @@ def human_readable_attr_name(name)
         bindable:                   'Plan bindable',
         plan_updateable:            'Plan updateable',
         schemas:                    'Plan schemas',
-        maximum_polling_duration:   'Maximum polling duration',
+        maintenance_info:           'Maintenance info',
+        maximum_polling_duration:   'Maximum polling duration'
```

Comment:
Should keep the trailing comma

Reply:
Hmm code climate was red because of this comma, so I removed it :/

Reply:
Fixed. I can swear codeclimate was red because of this comma, but now its fine. IDK what happened lat time.